command/node_status.go

```diff
@@ -443,6 +460,27 @@ func (c *NodeStatusCommand) outputTruncatedNodeDriverInfo(node *api.Node) string
 	return strings.Trim(strings.Join(drivers, ","), ", ")
 }
 
+func (c *NodeStatusCommand) outputNodeVolumeInfo(node *api.Node) {
+	c.Ui.Output(c.Colorize().Color("\n[bold]Host Volumes"))
+
+	size := len(node.HostVolumes)
+	nodeVolumes := make([]string, 0, size+1)
+
+	nodeVolumes = append(nodeVolumes, "Name|ReadOnly|Hidden|Source")
+
+	volumes := make([]string, 0, len(node.HostVolumes))
```

Comment:
I find the use of generic names here (e.g. `volumes` and `nodeVolumes` and `node.HostVolumes`) all for different concepts to make it a bit hard to follow.  Consider changing `volumes` to `volumeNames`, and `nodeVolumes` to `volumeInfo`.  Given that this function is all in context of volumes, I may even suggest dropping `volume` prefix.

Another thing that confused me was the transition from operating on `nodeVolumes` to operating on `volumes` then back to `nodeVolumes`.  I would suggest keeping all nodeVolumes work together so it's easier to understand its role and logic without the names detour.